mysql-test/suite/rocksdb/t/prefix_extractor_override.test

```diff
@@ -0,0 +1,93 @@
+--source include/have_rocksdb.inc
+
+CREATE TABLE t1 (id1 BIGINT, id2 BIGINT, id3 BIGINT, id4 BIGINT, PRIMARY KEY (id1, id2, id3, id4) comment 'cf1') ENGINE=rocksdb collate latin1_bin;
+--disable_query_log
+let $i = 1;
+while ($i <= 100) {
+  let $insert = INSERT INTO t1 VALUES(1, $i, $i, $i);
+  eval $insert;
+  inc $i;
+}
+--enable_query_log
+set global rocksdb_force_flush_memtable_now = 1;
+
+--echo
+--echo Original Prefix Extractor:
+--echo
+--sorted_result
+SELECT * FROM information_schema.rocksdb_cf_options WHERE option_type like '%prefix_extractor%';
+
+# BF used (4+8+8+8)
+select variable_value into @u from performance_schema.global_status where variable_name='rocksdb_bloom_filter_prefix_checked';
+SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
+select variable_value-@u from performance_schema.global_status where variable_name='rocksdb_bloom_filter_prefix_checked';
```

Comment:
assert?

Reply:
Not totally clear what you are suggesting here, maybe something like :

> --let $u1 = select variable_value from performance_schema.global_status where variable_name='rocksdb_bloom_filter_prefix_checked'
> SELECT COUNT(*) FROM t1 WHERE id1=1 AND id2=1 AND id3=1;
> --let $u2 = select variable_value from performance_schema.global_status where variable_name='rocksdb_bloom_filter_prefix_checked'
> --let $assert_text="Some rational statement of the problem"
> --let $assert_cond=$u1-$u2=1
> --source include/assert.inc

Reply:
Exactly this if you agree that "Some rational statement of the problem" helps to clarify the point of the check